include/godot_cpp/classes/wrapped.hpp

```diff
@@ -77,7 +77,7 @@ class Wrapped {
 		GDExtensionObjectPtr owner;
 		RecreateInstance *next;
 	};
-	inline static RecreateInstance *recreate_instance = nullptr;
+	static RecreateInstance *recreate_instance;
```

Comment:
Switching this one away from `inline static` is fine, but I have another PR (https://github.com/godotengine/godot-cpp/pull/1590 - still awaiting review) that removes it as part of fixing a different bug